src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -91,18 +91,22 @@ export interface ContentSection<BlockType> {
   content: BlockType[];
 }
 
-export const publicationSortOptions = [
-  'newest',
-  'oldest',
-  'alphabetical',
-] as const;
+export const publicationSortOptions = ['newest', 'oldest', 'title'] as const;
 
 export type PublicationSortOption = typeof publicationSortOptions[number];
 
-// TODO EES-3517 expand to include filters and search
-export interface ListPublicationsRequest {
+export type PublicationSortParam = 'published' | 'title' | 'relevance';
+
+export type PublicationOrderParam = 'asc' | 'desc';
+
+interface PublicationListRequest {
+  order?: PublicationOrderParam;
   page?: number;
-  sortBy?: PublicationSortOption;
+  pageSize?: number;
+  releaseType?: string;
```

Comment:
Think we could narrow the type constraints of `releaseType` here to be of `ReleaseType`

Reply:
Good spot, sorted now.